### PR TITLE
test_using_timestamp: update expected errors

### PR DESCRIPTION
**Pull request by bhalevy**
This mini-series updates the expected errors in `test/cql-pytest/test-timestamp.py`
to the ones changed in b7bbcdd178088c9fdfd217e9a419821d3a87c345.
Then, it renamed the test to `test_using_timestamp.py` so it would
run automatically with `test.py`.

### CODE REVIEW

File: test/cql-pytest/test-timestamp.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def restrict_future_timestamp():
 # test will need to be replaced by a test for the new feature instead of
 # expecting an error message.
 def test_key_writetime(cql, table1):
-    with pytest.raises(InvalidRequest, match='PRIMARY KEY part k'):
+    with pytest.raises(InvalidRequest, match='WRITETIME is not legal on partition key component k'):
```
Comment (nyh) [inline test/cql-pytest/test-timestamp.py:89]: If you change the error message check, this test will fail on Cassandra (please try `test/cql-pytest/run-cassandra test-timestamp.py::test_key_writetime`).
To allow this test to pass on both, you'll need to either check for two messages (e.g., `match='PRIMARY KEY part k|WRITETIME is not legal on partition key component k'` - it's a regex), or only check the common part (in this case it's only `' k'`... space and k...).

File: test/cql-pytest/test-timestamp.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def restrict_future_timestamp():
...
         cql.execute(f'SELECT writetime(k) FROM {table1}')
-    with pytest.raises(InvalidRequest, match='PRIMARY KEY part k'):
+    with pytest.raises(InvalidRequest, match='TTL is not legal on partition key component k'):
```
Comment (nyh) [inline test/cql-pytest/test-timestamp.py:91]: Same here.

**Review by nyh — CHANGES_REQUESTED** [review]

File: test/cql-pytest/test-timestamp.py (hunk 1 above)
Comment (bhalevy) [reply test/cql-pytest/test-timestamp.py:89]: Right.  I'll change to a regex using `|`.

**Comment by bhalevy** [comment]
In [b0bcad0](https://github.com/scylladb/scylladb/commit/b0bcad0c9105436606b07d39f837d6558c63875c):
* Extend the `match` regex to remain backward compatible with Cassandra

**Review by nyh — APPROVED** [review]

**Pull request merged**